packages/core/src/Tag/Tag.styles.tsx

```diff
@@ -5,15 +5,23 @@ import { outlineStyles } from "../utils/focusUtils";
 import { createClasses } from "../utils/classes";
 
 export const { staticClasses, useClasses } = createClasses("HvTag", {
-  root: {},
+  root: {
+    color: theme.colors.base_dark,
+
+    "& .MuiChip-avatar": {
```

Comment:
I think that it would be better to use the material exported classes object, something like : [& .${MuichipClasses.avatar}]